### ISSUE TITLE
Chromecast!

### ISSUE THREAD

**Issue by timiane** [rosenbjerg/NxPlx#25]
We have waited many long months for this feature, and have wept and cried many tears. We want! NO! WE DEMAND CHROMECAST SUPPORT!

**Comment by rosenbjerg** [comment]
Yes I want that too, but it seems hard to get working without a mpd file

**Comment by rosenbjerg** [comment]
Just do it
